checkov/secrets/git_history_store.py

```diff
@@ -71,24 +81,30 @@ def _add_new_secret(self, secret_key: str,
                                                                        'removed_commit_hash': ''})
                 return
         code_line = search_for_code_line(commit[secret.filename], secret.secret_value, secret.is_added)
-        self.secrets_by_file_value_type[secret_key].append(
-            {'added_commit_hash': commit_hash,
-             'removed_commit_hash': '',
-             'potential_secret': secret,
-             'code_line': code_line
-             })
+        enriched_potential_secret: EnrichedPotentialSecret = {
+            'added_commit_hash': commit_hash,
+            'removed_commit_hash': '',
+            'potential_secret': secret,
+            'code_line': code_line,
+            'created_by': commit.get(COMMIT_COMMITTER, ''),
+            'removed_date': '',
+            'create_date': commit.get(COMMIT_COMMITTED_DATETIME, '')
```

Comment:
would be weird to not have the committer and datetime here, could create debug log for now, if we don't have that info